{"name": "test_accept_order_without_id_order", "status": "failed", "statusDetails": {"message": "AssertionError: \u001b[31mFAILURE: \u001b[0mcheck 404 == 400\n\u001b[31mtest_accept_order.py\u001b[0m:65 in test_accept_order_without_id_order() -> check.equal(response_put.status_code, 400)\n\n\u001b[31mFAILURE: \u001b[0mcheck Not Found. == Недостаточно данных для поиска\n------------------------------------------------------------\nFailed Checks: 2", "trace": "\u001b[31mFAILURE: \u001b[0mcheck 404 == 400\n\u001b[31mtest_accept_order.py\u001b[0m:65 in test_accept_order_without_id_order() -> check.equal(response_put.status_code, 400)\n\n\u001b[31mFAILURE: \u001b[0mcheck Not Found. == Недостаточно данных для поиска\n------------------------------------------------------------\nFailed Checks: 2"}, "attachments": [{"name": "stdout", "source": "1382f782-acff-4ef0-a2fa-89a66d9fc2fa-attachment.txt", "type": "text/plain"}], "start": 1703083828739, "stop": 1703083829029, "uuid": "7c8c2113-b0a4-40b6-ab2d-6927ecfacace", "historyId": "8f15678ad05029a1e459df7528315815", "testCaseId": "8f15678ad05029a1e459df7528315815", "fullName": "tests.test_accept_order.TestGetOrder#test_accept_order_without_id_order", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestGetOrder"}, {"name": "host", "value": "inetshop-61"}, {"name": "thread", "value": "5368-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}